pom.xml

```diff
@@ -131,6 +132,18 @@
             <groupId>com.microsoft.azure.kusto</groupId>
             <artifactId>kusto-ingest</artifactId>
             <version>${kusto-sdk.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.core</groupId>
+                    <artifactId>jackson-databind</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
```

Comment:
In the PR , you can also run dependency:tree to see what versions of libs are in use.

Reply:
<img width="1137" alt="Screenshot 2022-08-11 at 4 20 52 PM" src="https://user-images.githubusercontent.com/108695755/184118056-964e4f3c-b3e0-45b7-bd1c-8e3cb6efb932.png">

Reply:
<img width="1480" alt="Screenshot 2022-08-11 at 4 40 42 PM" src="https://user-images.githubusercontent.com/108695755/184120901-78b1e27d-ccdd-47f5-a8e8-e69f64b66338.png">

Reply:
<img width="1054" alt="Screenshot 2022-08-11 at 4 44 27 PM" src="https://user-images.githubusercontent.com/108695755/184121358-2151374b-fa39-4176-b3de-31bd1e7a8f22.png">

Reply:
[mvn-dependency-tree-output.txt](https://github.com/Azure/kafka-sink-azure-kusto/files/9308168/mvn-dependency-tree-output.txt)